docs/specifications/feed.md

```diff
@@ -298,6 +298,7 @@ The type of the archive is given as a MIME type in the `type` attribute (since 0
 -   `application/x-ruby-gem` (`.gem`) - since version 1.0-rc1, not supported by Windows version
 -   `application/x-7z-compressed` (`.7z`) - Windows version only
 -   `application/x-msi` (`.msi`) - Windows version only
+-   `application/x-apple-diskimage` (`.dmg`)
```

Comment:
We should add "not supported by Windows version" here for now.

I'll look into how to add support for `.dmg` files on Windows, perhaps using 7-Zip.